hngi/fg-expense-backend
# [chore]: Init TravisCI and base route test

**What does this PR do?**

Initializes Travis CI and creates a dummy Jest test

**description of Task to be completed?**
- add travis.yml to configure Travis CI
- include coveralls for test coverage reporting
- add test for "/" base route

**How should this be manually tested?**
- run `npm run lint:fix` to ensure all lint errors are fixed
- run `npm run test` to run test
- create new PR to trigger build on Travis CI

**Any background context you want to provide?**
Temporarily running `npm run lint:fix` on CI because of existing lint errors, will be reverted to `npm run lint`. This should normally be ran locally before making PR

**What is the link to the user story on clubhouse if available?**
None yet
